### PR TITLE
Initial support for composer

### PR DESCRIPTION
**Pull request by phil-mm**
Adds initial support for composer

### CODE REVIEW

**Review by ryanbailey-fb — COMMENTED** [review]
Looks great!  All unity features seem to have corresponding counterparts.  Specifically happy to see the integration of ComposerActionDefaultHandler, ComposerSpeechDefaultHandler, & WitTtsSpeaker.  Nice work!

**Review by ryanbailey-fb — APPROVED** [review]

File: Source/Wit/Private/Wit/TTS/WitTtsService.cpp [hunk 1]
```diff
@@ -423,7 +423,7 @@ void UWitTtsService::OnVoicesRequestComplete(const TArray<uint8>& BinaryResponse
  * @param ErrorMessage [in] the error message
  * @param HumanReadableErrorMessage [in] longer human readable error message
  */
-void UWitTtsService::OnVoicesRequestError(const FString ErrorMessage, const FString HumanReadableErrorMessage)
+void UWitTtsService::OnVoicesRequestError(const FString& ErrorMessage, const FString& HumanReadableErrorMessage)
```
Comment (wangpingsx) [inline Source/Wit/Private/Wit/TTS/WitTtsService.cpp:426]: what is the reason to add "&"?  for memory efficiency?

**Comment by wangpingsx** [comment]
if you already changed it to not spawnable, then I think we can merge this

**Review by wangpingsx — APPROVED** [review]

File: Source/Wit/Private/Wit/TTS/WitTtsService.cpp (hunk 1 above)
Comment (phil-mm) [reply Source/Wit/Private/Wit/TTS/WitTtsService.cpp:426]: Yeah, passing by value instead of reference causes unnecessary copies of data.

**Pull request merged**